src/main/context-handler/context-handler.ts

```diff
@@ -152,9 +152,6 @@ export class ContextHandler implements ClusterContextHandler {
 
   protected async newApiTarget(timeout: number): Promise<httpProxy.ServerOptions> {
     const kubeAuthProxy = await this.ensureServerHelper();
-    const ca = this.resolveAuthProxyCa();
-    const clusterPath = this.clusterUrl.path !== "/" ? this.clusterUrl.path : "";
```

Comment:
Needing this seems a bit of a kludge anyway? The same check exists on line 134. Should it be removed too?